### ISSUE TITLE
SignIn Missing List Mark-Up 

### ISSUE THREAD

**Issue by johnhbenetech** [benetech/MathShare#789]
The two OAuth providers (Google and Microsoft) are not wrapped inside an unordered list to indicate their semantic grouping. 
 
Suggested Solution 
 
Wrap the two buttons which can be used to initiate the OAuth flows inside an unordered list element, with each button inside a < li> element instead of the current < div>. 
 
Once the advice from the previous “Missing Heading Hierarchy” section has been followed, a level 2 heading should be in place with a unique ID, just before the list mark-up begins.  Use that heading’s ID as the value of an aria-labelledby attribute on the list’s opening < ul> tag, as shown in the proposed code sample. 

Proposed Code Sample 
 
File: src/components/SignIn/index.js 
`<h2 id="signInServices" className={signIn.text}>Login using</h2> <ul className={signIn.buttonsContainer} aria-labelledby="signInServices">  <li>{this.renderGoogleBtn()}</li>  <li>{this.renderMicrosoftBtn()}</li> </ul>`

**Comment by jscholes** [comment]
Fixed in https://github.com/benetech/MathShare/commit/8f3887afdba20d8d46962c9ee6486cbaf3bd546c.
